src/vector/mod.rs

```diff
@@ -0,0 +1,5 @@
+//! This module contains all minimizers for functions with multiple arguments.
```

Comment:
Similarly, I would suggest something along the lines of "will minimize the function value across multiple dimensions"